qa/workunits/rados/test_envlibrados_for_rocksdb.sh

```diff
@@ -40,15 +19,18 @@ CURRENT_PATH=`pwd`
 ############################################
 # install prerequisites
 # for rocksdb
-case $(lsb_release -si) in
-	Ubuntu|Debian|Devuan)
+case $(distro_id) in
+	ubuntu|debian|devuan)
 		install g++ libsnappy-dev zlib1g-dev libbz2-dev librados-dev
 		;;
-	CentOS|Fedora|RedHatEnterpriseServer)
+	centos|fedora|rhel)
 		install gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel librados2-devel.x86_64
 		;;
+	opensuse*|suse|sles)
+		install gcc-c++ snappy-devel zlib-devel libbz2-devel libradospp-devel
```

Comment:
@smithfarm could you remove `libradospp-devel` from the list? as it does not exist in mimic.